src/sortStudents.ts

```diff
@@ -1,16 +1,53 @@
-
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
+}
+
+export type SortOrder = 'asc' | 'desc';
+
+function calcAverageGrade({ grades }: Student): number {
+  return grades.reduce((total, grade) => total + grade, 0) / grades.length;
 }
 
-// create SortOrder type
-export type SortOrder;
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  const studentsCopy = [...students];
+
+  return studentsCopy.sort((firstStudent, secondStudent): number => {
+    switch (sortBy) {
+      case SortType.Name:
+      case SortType.Surname:
+        return order === 'asc'
```

Comment:
cause we could've used here something like
```suggestion
        return order === SortType.Ascending
```
and to be dependant on plain strings